pages/settings/SettingsLog.qml

```diff
@@ -213,9 +213,27 @@ Rectangle {
         MoneroComponents.LineEdit {
             id: sendCommandText
             Layout.fillWidth: true
+            property var lastCommands: []
+            property int currentCommandIndex
             fontBold: false
             placeholderText: qsTr("command + enter (e.g 'help' or 'status')") + translationManager.emptyString
             placeholderFontSize: 16
+            Keys.onUpPressed: {
+                if (currentCommandIndex != 0) {
+                    sendCommandText.text = lastCommands[currentCommandIndex - 1]
+                    currentCommandIndex = currentCommandIndex - 1
+                }
+            }
+            Keys.onDownPressed: {
+                if (currentCommandIndex == lastCommands.length - 1) {
+                    currentCommandIndex = lastCommands.length;
+                    return text = "";
```

Comment:
?

Reply:
1) User enter values "zero", "one", "two", "three" that are stored in the array.
2) Use the up arrow to navigate to "three" (last index of array).
3) Pressing down arrow will display a blank line, which is the default behavior in a Ubuntu terminal. In a Windows prompt command, pressing down arrow would keep displaying "three", but I prefer Ubuntu behavior.

Reply:
Maybe write it like this
```
text = "";
return;
```